web/src/components/nomination/NameInput.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-row dense>
+    <v-col
+      cols="12"
+      md="3"
+      xs="12"
+    >
+      <v-text-field
+        ref="title"
+        :value="value.title"
+        label="Title"
+        outlined
+        @input="updateName('title', $event)"
+      />
+    </v-col>
+    <v-col
+      cols="12"
+      md="3"
+      xs="12"
+    >
+      <v-text-field
+        ref="first"
+        :value="value.first"
+        label="First Name"
+        outlined
+        @input="updateName('first', $event)"
+      />
+    </v-col>
+    <v-col
+      cols="12"
+      md="3"
+      xs="12"
+    >
+      <v-text-field
+        ref="middle"
+        :value="value.middle"
+        label="Middle Name"
+        outlined
+        @input="updateName('middle', $event)"
+      />
+    </v-col>
+    <v-col
+      cols="12"
+      md="3"
+      xs="12"
+    >
+      <v-text-field
+        ref="last"
+        :value="value.last"
+        label="Last Name"
+        outlined
+        @input="updateName('last', $event)"
+      />
```

Comment:
I'd rather have just one text field for full name and people can enter however they prefer. I don't see a big advantage of splitting first/middle/last name (what are we gonna do with all parts?), and some countries just don't have this structure.

(When I created a web app that had users from Indonesia, we had exactly this problem and changed the name input to a single text field because some people in Indonesia don't have a last name)